VIDEO_1_SCRIPT.md

```diff
@@ -0,0 +1,99 @@
+# Video 1: What are APIs
```

Comment:
<img width="600" alt="Screen Shot 2022-02-03 at 6 00 48 PM" src="https://user-images.githubusercontent.com/19964402/152460159-ecd48471-3f97-4add-bfda-ce98e8654f27.png">

Question, have you ever used Grammarly Pro to help check for plagiarism? 😀👍🏽👍🏽  I currently use it to help catch myself from accidentally writing something too similar from my research sources. It can feel hard to not write too similar sometimes, or at least, to me that is how it feels. 😅  So that is why I use Grammarly Pro because it also has a feature that helps me identify how closely my own writing may match that of someone else. 

Anyways, I ran it for this Video Script 1 so far (will do so for Video Script 2 ❤️)  and I actually dismissed a few of the flagged notes because I don't think they merited a check... that said, it currently shows a 5% plagiarism rate and I thought it would help to know about the flagged sections 😀👍🏽👍🏽 🥳

Reply:
Wow, this's super interesting! I use Grammarly but not the Premium version. I didn't know it had this tool that I find very useful. It is difficult for me to modify and rewrite the information when it is so technical 😅